app/presenters/etd_presenter.rb

```diff
@@ -0,0 +1,3 @@
+class ETDPresenter < Hyrax::WorkShowPresenter
```

Comment:
The convention seems to be EtdPresenter, you should rename this and the references to it.